### PR TITLE
Docs/tasks taskid

### PR DESCRIPTION
**Pull request by jstirnaman**
- list task runs (Closes https://github.com/influxdata/openapi/issues/418)
- delete task (Closes https://github.com/influxdata/openapi/issues/405)
- update task (Closes https://github.com/influxdata/openapi/issues/407)
- retry task (Closes https://github.com/influxdata/openapi/issues/392)
- manual task (Closes https://github.com/influxdata/openapi/issues/419)
- task run (Closes https://github.com/influxdata/openapi/issues/421)
- task logs (Closes https://github.com/influxdata/openapi/issues/422)
- miscellaneous style and grammar fixes in descriptions.

### CODE REVIEW

File: src/cloud/paths/tasks.yml [hunk 1]
```diff
@@ -147,14 +147,51 @@ post:
...
     Use this endpoint to create a scheduled task that runs a Flux script.
-    In your task, provide one of the following:
-
-      - _`flux`_ property with an inline Flux script
-      - _`scriptID`_ property with an [invokable script](#tag/Invokable-Scripts) ID
 
     #### InfluxDB Cloud
 
-      - Requires either _`flux`_ (Flux query) or _`scriptID`_ (invokable script ID) in the task.
+    - You can use either `flux` or `scriptID` to provide the task script.
```
Comment (sunbryely-influxdata) [inline src/cloud/paths/tasks.yml:153]:
```suggestion
    - You can use either use `flux` or `scriptID` to provide the task script.
```

Comment (jstirnaman) [reply src/cloud/paths/tasks.yml:153]: > - Requires either...

I'm trying to keep the platform/API the subject here as much as possible, e.g. "InfluxDB Cloud... requires...." Doesn't always work, though.

Comment (sunbryely-influxdata) [reply src/cloud/paths/tasks.yml:153]: sorry i think i read that sentence wrong, please disregard the comment

File: src/common/schemas/TelegramNotificationEndpoint.yml [hunk 2]
```diff
@@ -8,5 +8,5 @@
           description: Specifies the Telegram bot token. See https://core.telegram.org/bots#creating-a-new-bot .
           type: string
         channel:
-          description: ID of the telegram channel, a chat_id in https://core.telegram.org/bots/api#sendmessage .
+          description: The ID of the telegram channel; a chat_id in https://core.telegram.org/bots/api#sendmessage .
```
Comment (sunbryely-influxdata) [inline src/common/schemas/TelegramNotificationEndpoint.yml:11]:
```suggestion
          description: The ID of the telegram channel; a chat_id in https://core.telegram.org/bots/api#sendmessage.
```

**Review by sunbryely-influxdata — APPROVED** [review]

File: src/cloud/paths/tasks.yml (hunk 1 above)
Comment (jstirnaman) [reply src/cloud/paths/tasks.yml:153]: > sorry i think i read that sentence wrong, please disregard the comment

Well, I read the diff and your suggestion wrong when I made the comment.

**Pull request merged**